src/app/helpers/apiWhiteList.js

```diff
@@ -69,12 +69,12 @@ const urlChecks = {
    * @since - release-3.1.0
    */
   ROLE_CHECK: (resolve, reject, req, rolesForURL, REQ_URL) => {
-    logger.info({
```

Comment:
**logger.info** giving error and because of this we are not able to login, so comment this